### ISSUE TITLE
The widget returns the string but not list in the streamlit v0.83 above.

### ISSUE THREAD

**Issue by tonylegend** [gagan3012/streamlit-tags#30]
**Describe the bug**
The widget returns the string but not list in the streamlit v0.83 above.

**To Reproduce**
Just upgrade the streamlit to 0.83 above and using the example code in the repo can reproduce this issue.

**Expected behavior**
Return a list of strings like ['one', 'two', 'three']

**Expected behavior**
Return a string representation like "['one', 'two', 'three']".

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser Safari
 - Version 11.5.1

**Comment by GitHunter0** [comment]
@gagan3012 , same here, I will try to downgrade it

**Comment by gagan3012** [comment]
Hello,

I was testing this with 

```
streamlit==0.85.1
streamlit_tags==1.2.8
```

using the code 

```python
keywords = st_tags(
    label='# Enter Keywords:',
    text='Press enter to add more',
    value=['Zero', 'One', 'Two'],
    suggestions=['five', 'six', 'seven', 'eight', 'nine', 'three', 'eleven', 'ten', 'four'],
    maxtags=maxtags,
    key="aljnf")

st.write("### Type:")
st.write(type(keywords))
```

Output: `<class 'list'>`

This is the expected behavior from st_tags can you share your code too?

**Comment by GitHunter0** [comment]
@gagan3012 , I was testing with `streamlit 0.84.0`, the issue was solved after I updated to `streamlit 0.85.1`, thanks for the feedback!

**Comment by tonylegend** [comment]
> Hello,
> 
> I was testing this with
> 
> ```
> streamlit==0.85.1
> streamlit_tags==1.2.8
> ```
> 
> using the code
> 
> ```python
> keywords = st_tags(
>     label='# Enter Keywords:',
>     text='Press enter to add more',
>     value=['Zero', 'One', 'Two'],
>     suggestions=['five', 'six', 'seven', 'eight', 'nine', 'three', 'eleven', 'ten', 'four'],
>     maxtags=maxtags,
>     key="aljnf")
> 
> st.write("### Type:")
> st.write(type(keywords))
> ```
> 
> Output: `<class 'list'>`
> 
> This is the expected behavior from st_tags can you share your code too?

I did not try on 0.85 due to the dataframe dtype issue after pyarrow migration. Currently I can manually convert the results on 0.84. Thanks for your information anyway.
